hack/verify-docker-deps.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Add this to verify-all?

Reply:
Added

Reply:
Will add in a later CL. We need https://github.com/kubernetes/test-infra/pull/26312 to go through in order for Docker commands to be supported in verify-all.sh